vnext/ReactUWP/LocalByteCodeManager.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include "LocalByteCodeManager.h"
+#include "unicode.h"
+#include <winrt/Windows.Storage.h>
+
+#if _MSC_VER <= 1913
+// VC 19 (2015-2017.6) cannot optimize co_await/cppwinrt usage
+#pragma optimize( "", off )
+#endif
+
+using namespace winrt;
+
+namespace react { namespace uwp {
+
+std::future<winrt::Windows::Foundation::DateTime> LocalByteCodeManager::LoadCreatedDateTimeAsync()
+{
```

Comment:
We should add a capability to read bytecode from ms-appx:///bundle/app.bytecode (or whatever) to support the scenario where bytecode is generated during build